src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
-import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  moviesList: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ moviesList }) => (
+  <div className="movies">
+    {moviesList.map(movie => (
+      <MovieCard
+        key={movie.imdbId}
+        {...movie}
+      />
```

Comment:
this is a better way
```suggestion
        key={movie.imdbId}
        movie={movie}
      />
```